src/input/touchmouse.js

```diff
@@ -26,19 +33,13 @@ inherit(TouchMouseInput, Input, {
         var isTouch = (inputData.pointerType == INPUT_TYPE_TOUCH),
             isMouse = (inputData.pointerType == INPUT_TYPE_MOUSE);
```

Comment:
I think you can add the @rbyers suggested `MouseEvent.sourceCapabilites.firesTouchEvents` check in this. I guess this can be added at this place.
Something like this would do the trick
```js
if (isMouse && inputData.sourceCapabilites.firesTouchEvents) { 
   return;
}
```